partcad/src/partcad/part_config_manufacturing.py

```
@@ -0,0 +1,35 @@
+#
+# Licensed under Apache License, Version 2.0.
+#
+
+METHOD_NONE = None
+METHOD_ADDITIVE = 1
+
+
+class PartConfigManufacturing:
+    method: int | None
+
+    def __init__(self, final_config):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)